modules/jdbc-test/src/test/java/org/testcontainers/junit/SimplePostgreSQLTest.java

```diff
@@ -18,22 +19,36 @@
 public class SimplePostgreSQLTest {
 
     @Rule
-    public PostgreSQLContainer postgres = new PostgreSQLContainer();
+    public PostgreSQLContainer postgres = (PostgreSQLContainer) new PostgreSQLContainer().withCommand("postgres -c max_connections=42");
```

Comment:
Let's change this test, so the container is not managed by the JUnit rule but instead started/stopped manually inside each test method. 

This means you can easily have one explicit test with command override and one without. Check tests in other modules, such as kafka, to see an example of what I mean.

Reply:
Done.